docs/execution-providers/index.md

```diff
@@ -34,7 +34,7 @@ ONNX Runtime supports many different execution providers today. Some of the EPs
 |[Intel OpenVINO](../execution-providers/OpenVINO-ExecutionProvider.md)|[AMD MIGraphX](../execution-providers/MIGraphX-ExecutionProvider.md) (*preview*)|[ARM-NN](../execution-providers/ArmNN-ExecutionProvider.md) (*preview*)|
 ||[TVM](../execution-providers/TVM-ExecutionProvider.md) (*preview*)|[CoreML](../execution-providers/CoreML-ExecutionProvider.md) (*preview*)|
 ||[Intel OpenVINO](../execution-providers/OpenVINO-ExecutionProvider.md)|[TVM](../execution-providers/TVM-ExecutionProvider.md) (*preview*)|
-|||[Qualcomm SNPE](../execution-providers/SNPE-ExecutionProvider.md)
+||[AMD ROCm](../execution-providers/ROCm-ExecutionProvider.md)|[Qualcomm SNPE](../execution-providers/SNPE-ExecutionProvider.md)
```

Comment:
Shall we move this upward, so that it is layouted as

ROCm
MIGraphX

Reply:
AMD ROCm -> AMD ROCm (*preview*)

@faxu Can you confirm we are still in *preview* status?

Reply:
Should be preview - good catch.
After it's stabilized, we'll remove preview later, thx.
We'll evaluate it when rocm5.3 is released.

Reply:
Please list this under the AMD MIGraphX entry and add (Preview)